doc/man3/SSL_CONF_cmd.pod

```diff
@@ -211,6 +211,17 @@ that there will be no forward secrecy for the resumed session.
 enables strict mode protocol handling. Equivalent to setting
 B<SSL_CERT_FLAG_TLS_STRICT>.
 
+=item B<-no_anti_replay>
+
+Switches off replay protection. With replay protection on OpenSSL will
+automatically detect if a session ticket has been used more than once, TLSv1.3
+has been negotiated, and early data is enabled on the server. A full handshake
+is forced if a session ticket is used a second or subsequent time. Anti-Replay
+is on by default and is only used by servers. Anti-replay measures are required
+to comply with the TLSv1.3 specification. Some applications may be able to
+mitigate the replay risks in other ways and in such cases the built in OpenSSL
```

Comment:
Hyphenate "built in" to "built-in"

Reply:
I suggest 'this built-in '

Reply:
Done (although I stuck with "the" instead of "this")